src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -6579,14 +6619,18 @@ subroutine atm_bdy_adjust_dynamics_relaxzone_tend( config, tend, state, diag, me
          end if
 
       end do
+      !$acc end parallel
 
       !  Third (and last), the horizontal filter for ru
-
+      !$acc parallel default(present)
+      !$acc loop gang worker private(cell1, cell2, vertex1, vertex2, r_dc, r_dv, & 
+      !$acc    iCell, iVertex, invArea, iEdge_div, iEdge_vort, edge_sign, &
+      !$acc    laplacian_filter_coef, divergence1, divergence2, vorticity1, vorticity2)
       do iEdge = edgeStart, edgeEnd
 
          if ( (bdyMaskEdge(iEdge) > 1) .and. (bdyMaskEdge(iEdge) <= nRelaxZone) ) then ! relaxation zone
 
-            laplacian_filter_coef = dcEdge(iEdge)**2 * (real(bdyMaskEdge(iEdge)) - 1.)/   &
+            laplacian_filter_coef = dcEdge(iEdge)*dcEdge(iEdge)* (real(bdyMaskEdge(iEdge)) - 1.)/   &
```

Comment:
Pure curiosity about this change: could you explain your motivation a little? Does this have any effect?

Reply:
I was trying this out because I though it may be one of the obstacles to bit identical results - but probably not. Will remove this change. Although, I would also add that we could just be using the multiplication operator here instead of exponent. But that's for another time.

Reply:
Agreed, I think this may be better as a multiplication than exponent but that goes against keeping to a "light touch" in this phase.

Reply:
Compilers are reasonably good at spotting these opportunities for optimization. The `nvfortran` compiler turns this line:
```
pure function square(x)

    real, intent(in) :: x
    real :: square

    square = x**2

end function square
```
into this assembler:
```
	.text
	.globl	square_                         # -- Begin function square_
	.type	square_,@function
square_:                                # @square_
	vmovss	(%rdi), %xmm0                   # xmm0 = mem[0],zero,zero,zero
	vmulss	%xmm0, %xmm0, %xmm0
	retq
```